lib/compile.py

```diff
@@ -287,6 +291,7 @@ def compile_json(json_file):
         template = "default"
     script_contents = (script_templates[template] % script_desc)
 
-    new_script = open(json_file + '.py', 'w')
+    new_script = open(json_file + '.py', 'w', encoding='utf-8')
```

Comment:
This is fine, When it comes to encoding issues. Things are more complicated that they look. later on i will look into standardizing the entire code to work well with anything on any sys version (p2/p3)
my idea was to over write the open function in retriever where we do something line checking which version first and then using that open function through out.

But this will work for now